src/main/java/com/redhat/devtools/lsp4ij/launching/templates/LanguageServerTemplateManager.java

```diff
@@ -66,4 +76,123 @@ static Reader loadTemplateReader(@NotNull String path) {
         var is = LanguageServerTemplateManager.class.getClassLoader().getResourceAsStream(TEMPLATES_DIR + "/" + path);
         return is !=null? new InputStreamReader(new BufferedInputStream(is)) : null;
     }
-}
+
+    @Nullable
+    public LanguageServerTemplate importLsTemplate(@NotNull VirtualFile templateFolder) {
+        try {
+            return createLsTemplate(templateFolder);
+        } catch (IOException ex) {
+            LOGGER.warn(ex.getLocalizedMessage(), ex);
+        }
+        return null;
+    }
+
+    @Nullable
+    public LanguageServerTemplate createLsTemplate(VirtualFile templateFolder) throws IOException {
+        String templateJson = null;
+        String settingsJson = null;
+        String initializationOptionsJson = null;
+        String description = null;
+
+        for (VirtualFile file : templateFolder.getChildren()) {
+            if (file.isDirectory()) {
+                continue;
+            }
+            switch (file.getName()) {
+                case TEMPLATE:
+                    templateJson = VfsUtilCore.loadText(file);
+                    break;
+                case SETTINGS:
+                    settingsJson = VfsUtilCore.loadText(file);
+                    break;
+                case INITIALIZATION_OPTIONS:
+                    initializationOptionsJson = VfsUtilCore.loadText(file);
+                    break;
+                case README:
+                    description = VfsUtilCore.loadText(file);
+                    break;
+                default:
+                    break;
+            }
+        }
+
+        if (templateJson == null) {
+            // Don't continue, if no template.json is found
+            return null;
+        }
+        if (settingsJson == null) {
+            settingsJson = "{}";
+        }
+        if (initializationOptionsJson == null) {
+            initializationOptionsJson = "{}";
+        }
+
+        GsonBuilder builder = new GsonBuilder();
+        builder.registerTypeAdapter(LanguageServerTemplate.class, new LanguageServerTemplateDeserializer());
+        Gson gson = builder.create();
+
+        LanguageServerTemplate template = gson.fromJson(templateJson, LanguageServerTemplate.class);
+        template.setConfiguration(settingsJson);
+        template.setInitializationOptions(initializationOptionsJson);
+        if (StringUtils.isNotBlank(description)) {
+            template.setDescription(description);
+        }
+
+        return template;
+    }
+
+    public void exportLsTemplates(@NotNull VirtualFile exportZip, @NotNull List<LanguageServerDefinition> lsDefinitions) {

```

Comment:
Add Javadoc

This method could return a number of ls templates which has been exported. This number could be displayed when this method is called (don't display UI inside in this method, since we could write some tests with this method.